app/src/main/java/covid/trace/morocco/onboarding/RegisterNumberFragment.kt

```diff
@@ -121,16 +121,13 @@ class RegisterNumberFragment : OnboardingFragmentInterface() {
         }
 
         language.setOnClickListener {
-            context?.let{
```

Comment:
oh yeah calling `context?.let` is redundant here since context is never NULL

Reply:
this is old code